interface/usergroup/usergroup_admin.php

```diff
@@ -351,6 +351,7 @@
             "', mname = '"         . add_escape_custom(trim((isset($_POST['mname']) ? $_POST['mname'] : ''))) .
             "', lname = '"         . add_escape_custom(trim((isset($_POST['lname']) ? $_POST['lname'] : ''))) .
             "', suffix = '"         . add_escape_custom(trim((isset($_POST['suffix']) ? $_POST['suffix'] : ''))) .
+            "', google_signin_email = '" . add_escape_custom(trim((isset($_POST['google_signin_email']) ? $_POST['google_signin_email'] : ''))) .
```

Comment:
This could be written as
`add_escape_custom(trim($_POST['google_signin_email'] ?? ''))`
What you have is acceptable as well so, is this done?